tools/environments/barista-environment.prod.ts

```diff
@@ -0,0 +1,51 @@
+/**
+ * @license
+ * Copyright 2019 Dynatrace LLC
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { join } from 'path';
+import { isPublicBuild } from 'tools/util/is-public-build';
+
+/** The Barista project's root directory. */
+const ROOT_DIR = '/';
+
+/** Root directory of the iconpack repository. */
+const ICONS_ROOT = '/barista-icons/src';
+
+/** Path to the icons changelog relative to the iconpack root direcotry. */
+const ICONS_CHANGELOG = join(
+  ICONS_ROOT,
+  '../_build/barista-icons/_templates',
+  `CHANGELOG${isPublicBuild() ? '-public' : ''}.json`,
+);
+
+/** URL/IP where the Strapi CMS is located. */
+const STRAPI_ENDPOINT = process.env.STRAPI_ENDPOINT;
+
+/** Parts of internal URLs that should be removed on public build. */
+const INTERNAL_LINKS = process.env.INTERNAL_LINKS;
+
+export const environment = {
```

Comment:
Nit: This could implement a given interface that is the same for prod and default, to make it easier to keep them in sync.